### DISCUSSION TITLE
Can Renovate Tag a Commit

### DISCUSSION THREAD

**Discussion by IvanZenger** [renovatebot/renovate#21780] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on gitlab.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.58.2

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.9.4-ee

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

If Renovate updates a dependency and creates a merge request, it would be cool if there was an option to create a tag from it.
Or to Tag the Commit. 

In my case, I update a variable in my .gitlab-ci.yml file with the Regex manager (source: github-releases)

### What you tried so far.

Reading the Documentation. What i found is this: https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjop97qtsL-AhW_wAIHHf0_DUYQFnoECA4QAQ&url=https%3A%2F%2Fgithub.com%2Frenovatebot%2Frenovate%2Fissues%2F2928&usg=AOvVaw3da66VcTQSR8MeIcnfugWB

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Not currently supported or planned. You want Renovate to tag its own branch though? i.e. the MR commit?
